### ISSUE TITLE
Changing the client window size while connected prints Tracebacks on the server

### ISSUE THREAD

**Issue by mat128** [internap/fake-switches#78]
```
Unhandled Error
Traceback (most recent call last):
  File "/Users/mmitchell/projects/fake-switches/.tox/py27/lib/python2.7/site-packages/twisted/python/log.py", line 84, in callWithContext
    return context.call({ILogContext: newCtx}, func, *args, **kw)
  File "/Users/mmitchell/projects/fake-switches/.tox/py27/lib/python2.7/site-packages/twisted/python/context.py", line 118, in callWithContext
    return self.currentContext().callWithContext(ctx, func, *args, **kw)
  File "/Users/mmitchell/projects/fake-switches/.tox/py27/lib/python2.7/site-packages/twisted/python/context.py", line 81, in callWithContext
    return func(*args,**kw)
  File "/Users/mmitchell/projects/fake-switches/.tox/py27/lib/python2.7/site-packages/twisted/conch/ssh/channel.py", line 137, in requestReceived
    return f(data)
--- <exception caught here> ---
  File "/Users/mmitchell/projects/fake-switches/.tox/py27/lib/python2.7/site-packages/twisted/conch/ssh/session.py", line 96, in request_window_change
    self.session.windowChanged(winSize)
exceptions.TypeError: windowChanged() takes exactly 1 argument (2 given)
```

**Comment by lindycoder** [comment]
Oops wrong button :P

**Comment by mat128** [comment]
Closed in #96
